runtime-modules/bounty/src/tests/fixtures.rs

```diff
@@ -696,8 +568,8 @@ impl SubmitWorkFixture {
 
         let new_entry = Bounty::entries(self.bounty_id, self.entry_id);
 
-        if actual_result.is_ok() {
-            assert!(new_entry.work_submitted);
+        if actual_result.is_ok() && new_entry.is_some() {
```

Comment:
```suggestion
        if actual_result.is_ok() {
```
`actual_result.is_ok()` already implies that `new_entry.is_some()`